### PR TITLE
Update Download Clients

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
NO

#### Description
Updates default categories and help text for various download clients.

#### Todos
- [ ] Tests

### CODE REVIEW

File: src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public DelugeSettings()
             Host = "localhost";
             Port = 8112;
             Password = "deluge";
-            TvCategory = "tv-Lidarr";
+            TvCategory = "Lidarr";
```
Comment (majora2007) [inline src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs]: We should keep this consistent with other settings and change to lowercase.

Comment (Qstick) [inline src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs]: Will change Deluge, Rtorrent, and Utorrent to Lcase.

**Comment by Qstick** [comment]
Updated

**Comment by majora2007** [comment]
Looks good, merging.

**Pull request merged**